### PR TITLE
coverage: deflake another

### PR DESCRIPTION
**Pull request by alyssawilk**

### CODE REVIEW

**Comment by alyssawilk** [comment]
cc @KBaichoo

**Review by rojkov — APPROVED** [review]
LGTM!

**Review by jmarantz — APPROVED** [review]

**Comment by jmarantz** [comment]
format ...

File: test/server/guarddog_impl_test.cc [hunk 1]
```diff
@@ -820,6 +820,9 @@ TEST_P(GuardDogActionsTest, KillShouldTriggerGuardDogActions) {
   EXPECT_DEATH(die_function(), "ASSERT_GUARDDOG_ACTION");
 }
 
+
```
Comment (RyanTheOptimist) [inline test/server/guarddog_impl_test.cc:823]: To fix the format, can you remove this line?
```suggestion
```

**Pull request closed without merging**